rigex-front-end/src/components/createWellModal/wellModalStyles.scss

```diff
@@ -0,0 +1,50 @@
+@import '../../styles/colors.scss';
+
+button {
+    background-color: $base-white;
+    border: 0px;
+}
+
+.dialog-title {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 350px;
+    font-size: 1.6em;
+    margin-bottom: 15px;
+}
+
+.dialog-actions {
```

Comment:
If I traced your code correctly it looks like modal-container is the parent of all the rest of the classes therefore classes such as dialog-actions should be nested under it